roles/st2mistral/tasks/main.yml

```diff
@@ -1,18 +1,27 @@
 ---
+- name: Install st2mistral dependency
+  become: yes
+  package:
+    name: st2python
```

Comment:
Good catch :+1: 
I assume that's the case why mistral didn't work for you.

We install `st2python` as package dependency for `st2`, see: https://github.com/StackStorm/st2-packages/blob/a46cc2267ac4b23c8f21a3471824c7c7f430cd1a/packages/st2/rpm/st2.spec#L15

Seems like we should add the same dependency for `st2mistral` as well in `st2-packages`.